src/stores/business.ts

```diff
@@ -213,6 +214,270 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
     return !!requestedFiling
   }
 
+  /** Check if the specified action is allowed, else False */
+  const isAllowed = (action: AllowableActionE): boolean => {
```

Comment:
Is tis supposed to be in here or inside some util file ? I think we said we gona keep it in enumutils.ts ?

Reply:
I added a `enum-utilities.ts` that contains some helper functions to compare filling status. (BTW, it is a `EnumUtilities` class with static methods. But maybe we can just use functions instead of putting them inside a class)

For this `isAllowed` function, it is a wrapper for `isAllowedToFile()` so I think it is better to keep them in the same file.